kubeflow-pipelines/odh-ray-cluster/secret.enc.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: kubeflow-raydev-aws-secret
```

Comment:
does this get consumed by the worker image, specifically with this exact name?  technically its a misnomer as these should be s3 credentials, not aws (since this is reading from a Ceph cluster with s3 frontend, not amazon)

Reply:
with that said, if the image is looking for this exact secret name, not worth updating to fix

Reply:
Still in the process of trying to get the pods generated to use the secret; I'll do a quick force since it clearly is a misnomer.

Reply:
Actually might be better to do it with the changes in the next PR after I've figured out how this should work